SurveyTests/Domain/Use Case/UserSession/UserSessionUseCaseSpec.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UserSessionUseCaseSpec.swift
+//  SurveyTests
+//
+//  Created by Khanh on 16/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Nimble
+import Quick
+@testable import Survey
+
+final class UserSessionUseCaseSpec: QuickSpec {
+
+    override func spec() {
+        var useCase: UserSessionUseCaseMock!
```

Comment:
This test is testing `UserSessionUseCaseMock` and not `UserSessionUseCase`.
I suggest you DI KeychainProtocol in `UserSessionUseCase`. That way you can test `UserSessionUseCase` with `KeychainService`.